lib/fabrik/DirectorManager.js

```diff
@@ -92,7 +91,7 @@ class DirectorManager extends BaseManager {
   }
 
   getNetworks(index) {
-    return new Networks(infrastructure.networks, index, infrastructure.segmentation);
+    return new Networks(this.director.config.infrastructure.networks, index, this.director.config.infrastructure.segmentation);
```

Comment:
comments similar to [above](https://github.com/SAP/service-fabrik-broker/pull/7/files#r133368904)